File: LeetCode/961.py
#Link: https://leetcode.com/problems/n-repeated-element-in-size-2n-array/

class Solution:
    def repeatedNTimes(self, A: List[int]) -> int:
        counts = [0]*10000
        res = int()
        for x in A:
            counts[x] += 1
        for i in range(10000):
            if counts[i] == len(A)//2:
                res = i
                break
        return res
        

File: LeetCode/cpp/961.cpp
//Link: https://leetcode.com/problems/n-repeated-element-in-size-2n-array/

class Solution {
public:
    int repeatedNTimes(vector<int>& A) {
        int counts[10000] = {0}, res;
        for(auto num : A)
            counts[num]++;
        for(int i = 0 ; i < 10000 ; i++) {
            if(counts[i] == A.size()/2){
                res = i;
                break;
            }
        }
        return res;
    }
};
